redis/src/main/scala/zio/redis/Output.scala

```diff
@@ -290,15 +284,15 @@ object Output {
   }
 
   case object StreamOutput extends Output[Map[String, Map[String, String]]] {
-    protected def tryDecode(respValue: RespValue): Map[String, Map[String, String]] =
+    protected def tryDecode(respValue: RespValue)(implicit codec: Codec): Map[String, Map[String, String]] =
       respValue match {
         case RespValue.NullArray       => Map.empty[String, Map[String, String]]
         case RespValue.Array(entities) => extractMapMap(entities)
         case other                     => throw ProtocolError(s"$other isn't an array")
       }
   }
 
-  private def extractMapMap(entities: Chunk[RespValue]): Map[String, Map[String, String]] = {
+  private def extractMapMap(entities: Chunk[RespValue])(implicit codec: Codec): Map[String, Map[String, String]] = {
```

Comment:
Non-related, but let's rename to `extractNestedMap` :).

Reply:
Done